apps/central/src/components/admin/Admin.vue

```diff
@@ -6,7 +6,7 @@
       @cancel="showSigninForm = false"
     />
     <Spinner v-if="loading" />
-    <MessageError v-else-if="session.email !== 'admin'">
+    <MessageError v-else-if="!session.admin">
```

Comment:
I would prefer 'isAdmin' in this case so we know it is a boolean and not some string.